x-pack/plugin/esql/qa/single-cluster/src/yamlRestTest/java/org/elasticsearch/xpack/esql/qa/single_node/EsqlClientYamlIT.java

```diff
@@ -23,7 +23,7 @@ public class EsqlClientYamlIT extends ESClientYamlSuiteTestCase {
     @ClassRule
     public static ElasticsearchCluster cluster = ElasticsearchCluster.local()
         .distribution(DistributionType.DEFAULT)
-        .nodes(1)
+        .nodes(2)
```

Comment:
Won't this run us with two nodes all the time? I don't think we want that for the `single_node` cluster, right?

Reply:
Do you want to just run multi-node all the time? Or should we keep this at 1 and use a different test case with 2?